embedded-redis/src/main/java/com/playtika/test/redis/EmbeddedRedisBootstrapConfiguration.java

```diff
@@ -109,14 +114,35 @@ public GenericContainer redis(ConfigurableEnvironment environment,
         return redis;
     }
 
-    private void prepareRedisConfFiles(RedisProperties properties) throws Exception {
-        resolveTemplate(resourceLoader, "redis.conf", content -> content
+    private MountableFile prepareRedisConf() throws IOException {
+        return processTemplate("redis.conf", content -> content
                 .replace("{{requirepass}}", properties.isRequirepass() ? "yes" : "no")
                 .replace("{{password}}", properties.isRequirepass() ? "requirepass " + properties.getPassword() : "")
                 .replace("{{clustered}}", properties.isClustered() ? "yes" : "no")
                 .replace("{{port}}", String.valueOf(properties.getPort())));
-        resolveTemplate(resourceLoader, "nodes.conf", content -> content
+    }
+
+    private MountableFile prepareNodesConf() throws IOException {
+        return processTemplate("nodes.conf", content -> content
                 .replace("{{port}}", String.valueOf(properties.getPort()))
                 .replace("{{busPort}}", String.valueOf(properties.getPort() + 10000)));
     }
+
+    private MountableFile processTemplate(String templateName, UnaryOperator<String> templateProcessor) throws IOException {
```

Comment:
can you fix the logic inside `FileUtils` as it is used inside other containers as well? thanks

Reply:
Done